README.md

```diff
@@ -1,58 +1,47 @@
-## Getting Started with Create React App
+# Managed Simple Data Exchanger
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This repository contains the frontend code for the Managed Simple Data Exchanger written in React and Typescript.
 
-## Available Scripts
+The Managed Simple Data Exchanger application consists of
 
-In the project directory, you can run:
+- [managed-simple-data-exchanger-frontend](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend)
+- [managed-simple-data-exchanger-backend](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend)
 
-### `npm start`
+![Tag](https://img.shields.io/static/v1?label=&message=LeadingRepository&color=green&style=flat) The helm chart for installing the Managed Simple Data Exchanger is available in [managed-simple-data-exchanger](https://github.com/eclipse-tractusx/managed-simple-data-exchanger).
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+The Managed Simple Data Exchanger is designed to work with the [IAM](https://github.com/eclipse-tractusx/portal-iam).
 
-The page will reload if you make edits\
-You will also see any lint errors in the console.
+Note: if you'd like to know more about what Managed Simple Data Exchanger contains and what the application does, then please checkout the [user-guide](https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend/blob/main/docs/user-guide/README.md).
 
-### `npm test`
+## Run locally
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Here are the two ways to run the application locally on http://localhost:3000/
 
-### `npm build`
+### Local build & run
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+    npm install
+    npm run build
+    npm start
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Running the image from GitHub container registry
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+    export IMAGE=tractusx/sdefrontend:latest
+    docker pull $IMAGE
+    docker run --rm -d -p 3001:8080 --name sdefrontend $IMAGE
 
-## `npm eject`
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+## Notice for Docker image
```

Comment:
You're still missing some info for the docker image notice section:

- Link to your repository on GitHub
- Direct link to the Dockerfile used to build your image
- Link to LICENCE file in your repo as 'Project License' (make clear, that this is the PROJECT licence, not an image license)

Take [this as an example](https://github.com/eclipse-tractusx/item-relationship-service/blob/main/README.md#notice-for-docker-image).

Reply:
added the notice for docker.